README.md

```diff
@@ -9,6 +9,49 @@ This started as a fork of [Play-ReactiveMongo](https://github.com/ReactiveMongo/
 With some minimal effort, as the ReactiveMongo people had already done the majority of the work, we felt that adding a base repository class creates a library without some
 of the issues the other simpler libraries have.
 
+## Upgrading from 5.x.x to 6.x.x?
+
+With version 6.x.x of simple-reactivemongo, we are moving to the latest version of reactivemongo which comes with a few braking changes documented here:  
+http://reactivemongo.org/releases/0.12/documentation/release-details.html#breaking-changes
+
+You will most likely encounter some of the following issues. Please have a look on how we recommend to fix them.
+
+#### No Json serializer as JsObject found 
+
+Due to keeping parity with upstream, companion objects for inner classes do not have the ImplicitBSONHandlers handlers.
+If you see an error like this:
+```No Json serializer as JsObject found for type reactivemongo.bson.BSONDocument. Try to implement an implicit OWrites or OFormat for this type.```
+Try adding the following import
+```scala
+import reactivemongo.play.json.ImplicitBSONHandlers._
+```
+
+#### WriteResult is no longer an Exception
+
+The type hierarchy of the trait `WriteResult` has changed in new version of reactivemongo. 
+It’s no longer an Exception. As it now longer represents errors in the public API, the following properties have been removed: `errmsg`, `hasErrors`, `inError` and `message`.
```

Comment:
`As it no longer` instead of `As it now longer`

Reply:
`errmsg`, `hasErrors` and `inError` are now kept in protective scope and are internal state kept encapsulated. If you wish to know whether the command is in error or not, you would have to check for `ok`.